server/user-groups.ts

```diff
@@ -117,7 +117,16 @@ export abstract class Auth extends Map<ID, GroupSymbol | ''> {
 		const auth: Auth = room ? room.auth : Users.globalAuth;
 
 		const symbol = auth.getEffectiveSymbol(user);
-		let targetSymbol = (typeof target === 'string' || !target) ? target : auth.get(target);
+
+		let targetSymbol = target as GroupSymbol;
```

Comment:
`target` is actually not necessarily a `GroupSymbol` here. I think the previous cast was in the correct location.